writeaheadlog_test.go

```diff
@@ -15,6 +15,14 @@ import (
 	"github.com/NebulousLabs/fastrand"
 )
 
+// tempDir joins the provided directories and prefixes them with the testing
+// directory.
+func tempDir(dirs ...string) string {
```

Comment:
Maybe we should put `tempDir` and the other new helper functions into `build.go` so that they are all in one place?

Reply:
probably dependencies_test.go